Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Feed.kt

```diff
@@ -66,29 +80,48 @@ private fun Feed(
     }
 }
 
+@OptIn(ExperimentalAnimationApi::class)
 @Composable
 private fun SnackCollectionList(
     snackCollections: List<SnackCollection>,
     filters: List<Filter>,
     onSnackClick: (Long) -> Unit,
     modifier: Modifier = Modifier
 ) {
-    LazyColumn(modifier) {
-        item {
-            Spacer(Modifier.statusBarsHeight(additional = 56.dp))
-            FilterBar(filters)
-        }
-        itemsIndexed(snackCollections) { index, snackCollection ->
-            if (index > 0) {
-                JetsnackDivider(thickness = 2.dp)
+    var filtersVisible by rememberSaveable { mutableStateOf(false) }
+    Box {
+        LazyColumn(modifier) {
+
+            item {
+                Spacer(Modifier.statusBarsHeight(additional = 56.dp))
+                FilterBar(filters, onFilterClicked = { filtersVisible = true })
+            }
+            itemsIndexed(snackCollections) { index, snackCollection ->
+                if (index > 0) {
+                    JetsnackDivider(thickness = 2.dp)
+                }
+
+                SnackCollection(
+                    snackCollection = snackCollection,
+                    onSnackClick = onSnackClick,
+                    index = index
+                )
             }
-            SnackCollection(
-                snackCollection = snackCollection,
-                onSnackClick = onSnackClick,
-                index = index
-            )
         }
     }
+    AnimatedVisibility(
+        visible = filtersVisible,
+        enter = slideInVertically(
+            initialOffsetY = { -40 }
```

Comment:
Should this offset be in Dp? A constant pixel offset seems incorrect.

Reply:
I removed initial offset, it was unnecessary